src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralServiceTest.kt

```diff
@@ -635,8 +634,8 @@ class ReferralServiceTest @Autowired constructor(
       whenever(serviceProviderAccessScopeMapper.fromUser(user))
         .thenReturn(ServiceProviderAccessScope(setOf(serviceProviderFactory.create()), validContracts.toSet()))
 
-      val filteredReferrals = referralService.getSentReferralsForUser(user, null, null, null, null, null)
-      assertThat((filteredReferrals as List<Views.SentReferral>).size).isEqualTo(3)
+      val filteredReferrals = referralService.getSentReferralSummaryForUser(user, null, null, null, null, PageRequest.of(0, 10))
+      assertThat((filteredReferrals as Page<SentReferralSummary>).size).isEqualTo(10)
```

Comment:
Feels like this was testing that invalid contracts are properly filtered out (we get 3 referrals instead of 5), whereas the new test seems to test the page size -- is this correct?

Reply:
@sldblog , we noticed and updated this already